src/llmq/signing_shares.h

```diff
@@ -393,7 +412,7 @@ class CSigSharesManager : public CRecoveredSigsListener
     void ProcessMessage(const CNode* pnode, const std::string& strCommand, CDataStream& vRecv);
 
     void AsyncSign(const CQuorumCPtr& quorum, const uint256& id, const uint256& msgHash);
-    CSigShare CreateSigShare(const CQuorumCPtr& quorum, const uint256& id, const uint256& msgHash) const;
+    std::optional<CSigShare> CreateSigShare(const CQuorumCPtr& quorum, const uint256& id, const uint256& msgHash) const;
```

Comment:
probably needs `#include <optional>` like in #4781

Reply:
:D

Reply:
Thanks @UdjinM6. @PastaPastaPasta is always trying to break my builds :smirk: